mvpa2/base/constraints.py

```diff
@@ -299,25 +318,23 @@ def long_description(self):
 
     def short_description(self):
         cs = [c.short_description() for c in self.constraints
-                    if hasattr(c, 'short_description') and c.short_description() is not None]
+              if hasattr(c, 'short_description') and c.short_description() is not None]
         doc = ' or '.join(cs)
         if len(cs) > 1:
             return '(%s)' % doc
         else:
             return doc
 
 
-
 class Constraints(Constraint):
     """Logical AND for constraints.
-
     An arbitrary number of constraints can be given. They are evaluated in the
     order in which they were specified. The return value of each constraint is
     passed an input into the next. The return value of the last constraint
     is the global return value. No intermediate exceptions are caught.
-
```

Comment:
sorry, professor. Futurize automatically plays around with some of the spacing. I'll work on going through and making sure the spacing stays the same.